clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutor.scala

```diff
@@ -103,10 +103,14 @@ class DeleteExecutor[TI <: TransferIndex](deleteCommand: DeleteCommand[TI])
   )(implicit ec: ExecutionContext): Future[UpsertId] = {
     val pathsToDelete = existingMetadata.pathsToDelete.filter { path =>
       val pathExists = ioUtil.googleObjectExists(path)
-      if (!pathExists) {
-        logger.warn(
-          s"'$path' associated with $prettyKey does not exist in the cloud."
-        )
+      val err = s"'$path' associated with $prettyKey does not exist in the cloud."
+      (pathExists, deleteCommand.forceDelete) match {
+        case (false, false) =>
+          logger.error(err + " Use --force-delete to delete this file.")
```

Comment:
This is redundant since the rolled-up exception will get re-logged at the top level. I'd just throw the `err + " Use ..."`.